helm/easeprobe/templates/deployment.yaml

```diff
@@ -41,7 +41,8 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           ports:
-            - containerPort: 8181
+            - name: http
```

Comment:
Consider using Helm chart values to make the port name configurable. This would allow users to customize the port name if needed. [medium]